smriprep/workflows/outputs.py

```diff
@@ -73,137 +77,600 @@ def init_anat_reports_wf(*, freesurfer, output_dir, name="anat_reports_wf"):
-            input_names=["mask_file", "before", "after", "after_mask"],
-        ),
-        name="norm_msk",
-    )
-    norm_rpt = pe.Node(SimpleBeforeAfter(), name="norm_rpt", mem_gb=0.1)
-    norm_rpt.inputs.after_label = "Participant"  # after
-
-    ds_std_t1w_report = pe.Node(
+    ds_fsnative_t1w = pe.Node(
         DerivativesDataSink(
-            base_directory=output_dir, suffix="T1w", datatype="figures"
+            base_directory=output_dir,
+            mode="image",
+            to="T1w",
+            suffix="xfm",
+            extension="txt",
+            **{"from": "fsnative"},
         ),
-        name="ds_std_t1w_report",
+        name="ds_fsnative_t1w",
         run_without_submitting=True,
     )
 
     # fmt:off
     workflow.connect([
-        (inputnode, tf_select, [(('template', _drop_cohort), 'template'),
-                                (('template', _pick_cohort), 'cohort')]),
-        (inputnode, norm_rpt, [('template', 'before_label')]),
-        (inputnode, norm_msk, [('std_t1w', 'after'),
-                               ('std_mask', 'after_mask')]),
-        (tf_select, norm_msk, [('t1w_file', 'before'),
-                               ('brain_mask', 'mask_file')]),
-        (norm_msk, norm_rpt, [('before', 'before'),
-                              ('after', 'after')]),
-        (inputnode, ds_std_t1w_report, [
-            (('template', _fmt), 'space'),
-            ('source_file', 'source_file')]),
-        (norm_rpt, ds_std_t1w_report, [('out_report', 'in_file')]),
+        (inputnode, lta2itk, [('fsnative2t1w_xfm', 'in_xfms')]),
+        (inputnode, ds_t1w_fsnative, [('source_files', 'source_file')]),
+        (lta2itk, ds_t1w_fsnative, [('out_inv', 'in_file')]),
+        (inputnode, ds_fsnative_t1w, [('source_files', 'source_file')]),
+        (lta2itk, ds_fsnative_t1w, [('out_xfm', 'in_file')]),
+        (ds_fsnative_t1w, outputnode, [('out_file', 'fsnative2t1w_xfm')]),
+        (ds_t1w_fsnative, outputnode, [('out_file', 't1w2fsnative_xfm')]),
     ])
     # fmt:on
-
-    if freesurfer:
-        from ..interfaces.reports import FSSurfaceReport
-
-        recon_report = pe.Node(FSSurfaceReport(), name="recon_report")
-        recon_report.interface._always_run = True
-
-        ds_recon_report = pe.Node(
-            DerivativesDataSink(
-                base_directory=output_dir, desc="reconall", datatype="figures"
-            ),
-            name="ds_recon_report",
-            run_without_submitting=True,
-        )
-        # fmt:off
-        workflow.connect([
-            (inputnode, recon_report, [('subjects_dir', 'subjects_dir'),
-                                       ('subject_id', 'subject_id')]),
-            (recon_report, ds_recon_report, [('out_report', 'in_file')]),
-            (inputnode, ds_recon_report, [('source_file', 'source_file')])
-        ])
-        # fmt:on
-
     return workflow
 
 
-def init_anat_derivatives_wf(
+def init_anat_second_derivatives_wf(
```

Comment:
This could definitely stand refactoring, but I'd prefer to keep that until later.